README.md

```diff
@@ -273,7 +273,9 @@ Mach. Intell. Res., 19:412-424, 2022.
 - <a name="Gehrig22arxiv"></a>Gehrig, D., Scaramuzza, D.,  
 *[Are High-Resolution Cameras Really Needed?](https://arxiv.org/abs/2203.14672)*,  
 arXiv, 2022. [YouTube](https://youtu.be/HV9_FhS-f88), [code](https://uzh-rpg.github.io/eres/).  
-
+- <a name="Ercan23cvprw"></a>Ercan, B., Eker, O., Erdem, A., Erdem, E.  
+*[EVREAL: Towards a Comprehensive Benchmark and Analysis Suite for Event-based Video Reconstruction](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Ercan_EVREAL_Towards_a_Comprehensive_Benchmark_and_Analysis_Suite_for_Event-Based_CVPRW_2023_paper.pdf)*,  
+IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2023. [PDF](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Ercan_EVREAL_Towards_a_Comprehensive_Benchmark_and_Analysis_Suite_for_Event-Based_CVPRW_2023_paper.pdf), [Project Page](https://ercanburak.github.io/evreal.html), [Suppl.](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/supplemental/Ercan_EVREAL_Towards_a_CVPRW_2023_supplemental.zip), [Code](https://github.com/ercanburak/EVREAL).
```

Comment:
Thanks. BTW, [33] has open-source code :) https://github.com/tub-rip/event_based_image_rec_inverse_problem 
which is omitted in Table 1.

Reply:
Ah, thank you very much for bringing that comment here :) Let me respond and hopefully clarify that now: 

In Table 1, what we indicate with the "open source" column is whether that work has its "evaluation" code/setup open, meaning that one can run a script and generate the quantitative results in that work, similar to [this](https://github.com/RodrigoGantier/SPADE_E2VID/blob/master/benchmark.py) and [that](https://github.com/WarranWeng/ET-Net/blob/master/scripts/eval.py). I think being able to reproduce quantitative results is important since it is not feasible for one to confirm that the code worked as expected just by looking at the qualitative results. I believe some years-old issues on code repositories of earlier works (examples: [1](https://github.com/uzh-rpg/rpg_e2vid/issues/19), [2](https://github.com/TimoStoff/event_cnn_minimal/issues/22), [3](https://github.com/tudelft/ssl_e2vid/issues/2)) indicate that this is indeed something needed in the community. 

Given that clarification, please let us know if we are still missing something in Table 1, particularly related to [33] :)

Reply:
Thanks for the clarification. I think it is a matter of my expectations: when I read "Open Source" in the column header and there is one method/reference per row, I thought it was about listing the references that had the code available vs those that did not. For example, the first row [27] has no code available. Maybe a header "Eval. code available" would have been better. No worries.

Reply:
Thank you very much for your comments, you are right, it could have been explained better (if we weren't trying so hard to fit into the page limit).

Looking forward to the workshop!